pom.xml

```diff
@@ -346,6 +352,18 @@
           </exclusion>
         </exclusions>
       </dependency>
+      <dependency>
+        <groupId>org.slf4j</groupId>
+        <artifactId>log4j-over-slf4j</artifactId>
+        <version>${slf4j.version}</version>
+        <scope>test</scope>
```

Comment:
Since we don't know what kind of logging library will be used on the cluster, we only change it for test scope.